README.md

```diff
@@ -593,6 +593,16 @@ Processing can be enabled for a single version by setting the processing flag on
 @uploader.thumb.enable_processing = true
 ```
 
+## Fog
+
+If you want to use fog you must add in your CarrierWave initializer the
+following lines
+
+```ruby
+config.fog_provider = 'fog'
```

Comment:
I believe this line is not entirely necessary. `'fog'` is the default so the user might need to specify the config only if it using an extracted provider, like `fog-aws` or `fog-google`.

Perhaps we could add a note about this?